functions-email-markup.php

```diff
@@ -286,3 +286,31 @@ function gmucf_email_markup( $content ) {
 		}
 	}
 }
+
+
+/**
+ * Convert paragraphs into email friendly tables
+ *
+ * @since 2.1.0
+ * @author RJ Bruneel
+ * @param array $content Contains the email content.
+ */
+function convert_email_markup( $content ) {
+
+	$table_open = '
```

Comment:
Using output buffers for multiline strings like these could make them a little easier to read + edit.  e.g.
```php
function convert_email_markup( $content ) {
ob_start();
?>
<table class="">
    ...
</table>
<?php
$table_open = ob_get_clean();
...
```